src/marqo/tensor_search/add_docs.py

```diff
@@ -88,18 +90,23 @@ def threaded_download_and_preprocess_content(allocated_docs: List[dict],
                 if field not in tensor_fields:
                     continue
                 if isinstance(doc[field], str) or force_download:
-                    modality = infer_modality(doc[field])
-                    if (modality == Modality.IMAGE and is_no_index) or (
-                        modality == Modality.IMAGE and is_unstructured_index) or (
-                        is_structured_index and media_field_types_mapping[field] == FieldType.ImagePointer):
-
-                        if (marqo_index is not None
-                                and marqo_index.model.properties.get('type') in [ModelType.LanguageBind]
-                                and marqo_index.model.properties.get('supported_modalities') is not None
-                                and Modality.IMAGE not in marqo_index.model.properties.get('supported_modalities')):
+                    try:    
+                        inferred_modality = infer_modality(doc[field])
```

Comment:
Doest this mean we are still inferring modality for the image field? We shouldn't do this to avoid regression.